### ISSUE TITLE
Better handling of file name duplicates

### ISSUE THREAD

**Issue by WolfgangDpunkt** [danburzo/percollate#144]
This also concerns the conversion of a large number of documents at once.
I have noticed that when Percollate finds the identical title in several files, it results in trying to save all the files under the identical file name, thus overwriting each other.
So if there are 50 html files with the same title, it ends up saving only one epub file.
Percollate should (optionally) rename duplicate files, for example by numbering them.

How can duplicate titles occur?
I could observe that this happens when the "og:title" is incorrectly set by an editor to the general name of the publication, not to the specific article title.
```
<meta property="og:title" content="newspaper name" />
<title>article name</title>
```

My manual solution was to remove the og:title everywhere by search/replace, then Percollate took the article name for the file name and everything went correctly.

Thank you for the attention to this matter, I appreciate it very much.

**Comment by danburzo** [comment]
Hi @WolfgangDpunkt, thanks for logging this issue. I've created the PR above to address this issue by appending `-1`, `-2` etc. to overlapping filenames.

**Comment by danburzo** [comment]
The fix has been published in `percollate@2.2.2`.

**Comment by WolfgangDpunkt** [comment]
Dear @danburzo ,

wow, that's a really quick solution! 
Thank you very much, that's very kind.
I will test the new version as soon as possible.
Best regards
Wolfgang
